### PR TITLE
Fix upload on Travis.

### PR DESCRIPTION
**Pull request by pelson**

### CODE REVIEW

File: ci_support/upload_or_check_non_existence.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def upload(cli, meta, owner, channels):
     try:
         with open('binstar.token', 'w') as fh:
             fh.write(cli.token)
-        subprocess.check_call(['anaconda-client', '--quiet', '-t', 'binstar.token',
```
Comment (ocefpaf) [inline ci_support/upload_or_check_non_existence.py:47]: Renamed again :unamused:

Comment (pelson) [inline ci_support/upload_or_check_non_existence.py:47]: Oh yeah... thanks @ocefpaf. I'm getting there with pulling this into shape.

Comment (pelson) [inline ci_support/upload_or_check_non_existence.py:47]: P.S. Hopefully it is obvious why I couldn't put Obvious-CI into the staged-recipes repo (ah don't you love obvious recursiveness :stuck_out_tongue_winking_eye:)

Comment (ocefpaf) [inline ci_support/upload_or_check_non_existence.py:47]: :smile:

**Pull request merged**